AIDevGallery.SourceGenerator/Models/Model.cs

```diff
@@ -24,4 +24,7 @@ internal class Model
     [JsonConverter(typeof(SingleOrListOfStringConverter))]
     [JsonPropertyName("FileFilter")]
     public List<string>? FileFilters { get; init; }
+    public List<AIToolkitAction>? AIToolkitActions { get; set; }
+    public string? AIToolkitId { get; set; }
+    public string? AIToolkitFinetuningId { get; set; }
```

Comment:
Is this property being used at all? Or will it be used in the future? Seems like you would use one or the other only, no? So maybe it can be simplified to a single property.

Reply:
It will be used in the future once they add the finetuning deeplink - thought I'd just get ahead of it but can remove if necessary.

Reply:
If we don't know when that is coming, I would remove it, but if it is a follow up PR that is coming soon, I'm ok with keeping it.